scripts/casefold_db.py

```diff
@@ -101,7 +102,6 @@ def sendEmailWithBackoff(
             template_file = sydent.get_branded_template(
                 None,
                 "migration_template.eml",
-                ("email", "email.template"),
             )
```

Comment:
Yes, that was deliberate as `email.template` is used elsewhere for the verification email template. I thought preventing accidentally sending lots of verification emails instead of the "you don't have to do anything" migration emails made more sense!